services/dispatcher/dispatcher.cpp

```diff
@@ -438,7 +428,26 @@ namespace services::dispatcher {
     }
 
     void dispatcher_t::wal_success(components::session::session_id_t& session, services::wal::id_t wal_id) {
+        trace(log_, "dispatcher_t::wal_success session : {}, wal id: {}", session.data(), wal_id);
         actor_zeta::send(manager_disk_, dispatcher_t::address(), disk::handler_id(disk::route::flush), session, wal_id);
+
+        if(!is_session_exist(session_to_address_, session)){
+            return;
+        }
+        
+        auto session_obj = find_session(session_to_address_, session);
+        const bool is_from_wal = session_obj.address().get() == manager_wal_.get();
+        if(is_from_wal){
+            return;
```

Comment:
Need to clean session`s storage:
```
remove_session(session_to_address_, session);
```

Reply:
`ottergon/integration/cpp/test/test_save_load.cpp:167: FAILED:`

Reply:
@makeyavelly